addons/sys_rack/fnc_addRackOnReturn.sqf

```diff
@@ -16,14 +16,18 @@
  */
 #include "script_component.hpp"
 
-params ["_vehicle","_rackId"];
+params ["_vehicle", "_rackId"];
 
-private _rack = _rackId createVehicle [0,0,0];
+private _rack = _rackId createVehicle [-1000,-1000,-1000];
```

Comment:
Should use Alternative Syntax `createVehicle [type, position, markers, placement, special]` with `CAN_COLLIDE` to make sure `nearObjects` will pick it up in case other objects are there. The `nearObjects` syntax used below also uses PositionAGL so you don't have to transform `ATLToASL` then.